app/models/manageiq/providers/hawkular/middleware_manager/event_catcher/runner.rb

```diff
@@ -0,0 +1,78 @@
+require "manageiq/providers/hawkular/middleware_manager/event_catcher/monitor"
```

Comment:
I also did not think it should be necessary.  But it did not work without it, Instead it seemed to load a different Monitor, despite that I use a name-spaced class reference.  Without the require this .new failed with invalid # or arguments 1 for 0:

```
  def event_monitor_handle
    @event_monitor_handle ||= ManageIQ::Providers::Hawkular::MiddlewareManager::EventCatcher::Monitor.new(@ems)
  end
```